scripts/undocumented_classes.txt

```diff
@@ -50,4 +50,5 @@ com.mapbox.navigation.ui/OnNavigationReadyCallback
 com.mapbox.navigation.ui/RecenterBtnClickListener
 com.mapbox.navigation.ui/RouteOverviewBtnClickListener
 com.mapbox.navigation.ui/SoundButton
-com.mapbox.navigation.ui/SummaryBottomSheetCallback
\ No newline at end of file
+com.mapbox.navigation.ui/SummaryBottomSheetCallback
+com.mapbox.navigation.ui.speed/SpeedLimitView
```

Comment:
nit: move line `com.mapbox.navigation.ui.speed/SpeedLimitView` upper to alphabetical sequence 🙏